### DISCUSSION TITLE
Why was I subscribed to 'github/copilot-preview' without my consent?

### DISCUSSION THREAD

**Discussion by ryanobjc** [community/community#6643] [Copilot Conversations]
All of a sudden I started to get emails, and many other people as well.

Is this the variation of "no such thing as bad news" ?

**Comment by ryanobjc** [comment]
So I did get an email I finally found: "You were automatically subscribed because you’ve been given access to the repository."

Yet the repo is public. What gives? What's going on?

**Reply by davecheney** [reply]
Sorry for the spam. Earlier today the repo was not public but required 2fa to be configured to access it which was a burden for some users. We have since made the repo public to remove the 2fa requirement.

You can find more details here: https://github.com/github/feedback/discussions/6632
